### PR TITLE
Configurable delay between password reset

### PR DESCRIPTION
**Pull request by aeris**

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -253,6 +255,10 @@ func GetConfig() *Config {
 	return config
 }
 
+func PasswordResetInterval() time.Duration {
```
Comment (nono) [inline pkg/config/config.go]: Travis build failed on the lint step. Exported functions (those with a name that starts by an upper case letter) must have a comment. Can you fix it?

**Comment by jinroh** [comment]
Also you should pass a `go fmt ./...`  command to format the code.

File: pkg/config/config.go [hunk 2]
```diff
@@ -253,6 +255,11 @@ func GetConfig() *Config {
 	return config
 }
 
+// PasswordResetInterval returns the minimal delay between two password reset
+func PasswordResetInterval() time.Duration {
+	return config.PasswordResetInterval
```
Comment (jinroh) [inline pkg/config/config.go]: You could make this method return a default like that, for instance:

```go
func PasswordResetInterval() time.Duration {
	const defaultPasswordResetInterval = 15 * time.Minute
	if config.PasswordResetInterval == 0 {
		return defaultPasswordResetInterval
	}
	return config.PasswordResetInterval
}
```

File: pkg/config/config.go [hunk 3]
```diff
@@ -253,6 +255,11 @@ func GetConfig() *Config {
...
+}
+
 // Setup Viper to read the environment and the optional config file
 func Setup(cfgFile string) (err error) {
 	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
```
Comment (jinroh) [inline pkg/config/config.go]: You can add a 

```go
viper.SetDefault("password_reset_interval", 15 * time.Minute)
```

**Comment by jinroh** [comment]
Did a followup of this PR in #1160. It was failing due to malfunctions in our configuration defaults not applied for tests.

**Pull request closed without merging**